fecgov/fec-cms
# Update candidates browse data page

## Summary (required)

- Resolves #4778 

Add raising by the numbers cards under house and senate section, moved presidential learn more link within presidential exports accordion, updated presidential candidate link to 2024

### Required reviewers

1 UX & 1 frontend

## Impacted areas of the application

General components of the application that this PR will affect:

-  Browse data candidates tab: http://localhost:8000/data/browse-data/?tab=candidates

## Screenshots

![Screen Shot 2021-08-10 at 12 56 04 PM](https://user-images.githubusercontent.com/12799132/128902685-b979f6cf-3176-417e-8b4a-028c5e216295.png)

## How to test

- `npm run build-sass` for good measure, I didn't update any styles this time around
- `./manage.py runserver`
- Check that the browse data candidates tab looks like the comp in the [issue](#4778 )
